source/main/ipc/getHardwareWalletChannel.ts

```diff
@@ -76,6 +77,19 @@ class EventObserver {
           if (!devicesMemo[device.path]) {
             logger.info('[HW-DEBUG] CONSTRUCTOR ADD');
 
+            const walletData: LedgerDevicePayload = {
+              disconnected: false,
+              deviceType: 'ledger',
+              deviceId: null,
+              // Available only when Cardano APP opened
+              deviceModel: deviceModel?.id || '',
+              // e.g. nanoS
+              deviceName: deviceModel?.productName || '',
+              // e.g. Test Name
+              path: device?.path || '',
+              product: device?.product || '',
```

Comment:
We should use native types from the [ledger](https://github.com/LedgerHQ/ledgerjs/blob/23ba9fa883db9f49f81de236c9232df2c5ca1020/packages/devices/src/index.ts#L197) library instead of creating our ones. I guess this was done when we had flow and we couldn't simply import typescript types

Reply:
https://input-output.atlassian.net/browse/DDW-1093